lib/v2/verse/articles.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const asyncPool = require('tiny-async-pool');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category;
+    const currentUrl = `https://www.verse.com.tw/articles/${category}/`;
```

Comment:
You are visiting `https://www.verse.com.tw/articles/undefined/` if `category` is not present.
log: https://github.com/DIYgod/RSSHub/actions/runs/3703115344/jobs/6274175147#step:11:15